sdk/storage/azblob/appendblob/client.go

```diff
@@ -255,16 +255,6 @@ func (ab *Client) SetLegalHold(ctx context.Context, legalHold bool, options *blo
 	return ab.BlobClient().SetLegalHold(ctx, legalHold, options)
 }
 
-// SetTier operation sets the tier on a blob. The operation is allowed on a page
-// blob in a premium storage account and on a block blob in a blob storage account (locally
-// redundant storage only). A premium page blob's tier determines the allowed size, IOPS, and
-// bandwidth of the blob. A block blob's tier determines Hot/Cool/Archive storage type. This operation
-// does not update the blob's ETag.
-// For detailed information about block blob level tiering see https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-storage-tiers.
-func (ab *Client) SetTier(ctx context.Context, tier blob.AccessTier, o *blob.SetTierOptions) (blob.SetTierResponse, error) {
```

Comment:
Was this released already?

Reply:
"The Set Blob Tier operation sets the access tier on a blob. The operation is allowed on a page blob in a premium storage account and on a block blob in a blob storage or general purpose v2 account. "
Appears its not supported on AppendBlob as such.

Reply:
This is unfortunate.  My suggestion is to add a `// Deprecated: <message>` comment explaining this instead of removing it.

Reply:
I can add a comment instead of removing it if that is the general consensus.

Reply:
I'd also remove all the doc comments (you might have to keep a small `// SetTier` doc comment) and have the API return an error stating that this API doesn't work for append blobs.